test/testdata/addPackageTest/cabal-exe/.hie-bios

```diff
@@ -0,0 +1 @@
+cabal-helper-helper . $1
```

Comment:
should we remove all these? AFAIK, the tests dont use them, currenlty

Reply:
Yeah, we need to remove them.